pip_review/__main__.py

```diff
@@ -49,24 +50,76 @@ def check_output(*args, **kwargs):
 '''
 
 # parameters that pip list supports but not pip install
-LIST_ONLY = set('l local path format not-required exclude-editable include-editable'.split())
+LIST_ONLY = {
+    'l',
+    'local',
+    'path',
+    'format',
+    'not-required',
+    'exclude-editable',
+    'include-editable',
+}
 
 # parameters that pip install supports but not pip list
-INSTALL_ONLY = set('c constraint no-deps t target platform python-version implementation abi root prefix b build src U upgrade upgrade-strategy force-reinstall I ignore-installed ignore-requires-python no-build-isolation use-pep517 install-option global-option compile no-compile no-warn-script-location no-warn-conflicts no-binary only-binary prefer-binary no-clean require-hashes progress-bar'.split())
+INSTALL_ONLY = {
+    'c',
+    'constraint',
+    'no-deps',
+    't',
+    'target',
+    'platform',
+    'python-version',
+    'implementation',
+    'abi',
+    'root',
+    'prefix',
+    'b',
+    'build',
+    'src',
+    'U',
+    'upgrade',
+    'upgrade-strategy',
+    'force-reinstall',
+    'I',
+    'ignore-installed',
+    'ignore-requires-python',
+    'no-build-isolation',
+    'use-pep517',
+    'install-option',
+    'global-option',
+    'compile',
+    'no-compile',
+    'no-warn-script-location',
+    'no-warn-conflicts',
+    'no-binary',
+    'only-binary',
+    'prefer-binary',
+    'no-clean',
+    'require-hashes',
+    'progress-bar',
+}
```

Comment:
Also, does a constant really need to occupy 37 lines just because it is cleaner?